Controllers/HomeController.cs

```diff
@@ -3,21 +3,90 @@
 using System.Linq;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
+using System.Text.RegularExpressions;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using Newtonsoft.Json;
 
 namespace Zoltu.Bags.Client.Controllers
 {
 	public class HomeController : Controller
 	{
+		// Best practices
+		// https://www.asp.net/web-api/overview/advanced/calling-a-web-api-from-a-net-client
+		static HttpClient client = new HttpClient();
+
 		[Route("")]
 		[Route("app/{*path}")]
-		public IActionResult App(string path = null)
+		public IActionResult App(String path = null)
 		{
-			ViewData["url"] = "https://bagcupid.com/";
-			ViewData["type"] = "website";
-			ViewData["title"] = "Bag Cupid";
-			ViewData["description"] = "What is your dream bag? Are you having trouble finding it? Let us help you!";
+			String url = "https://bagcupid.com/";
+			String type = "website";
+			String title = "Bag Cupid";
+			String description = "What is your dream bag? Are you having trouble finding it? Let us help you!";
+			String image = "https://bagcupid.com/img/logo/bagcupid.png";
+
+			if (!String.IsNullOrEmpty(path))
+			{
+				String ProductId = MatchRegexAndReturnValue(path, @"product\/(\d+)");
+				String TagId = MatchRegexAndReturnValue(path, @"tags\/(.*?)\/*?$");
+				String MinPrice = MatchRegexAndReturnValue(path, @"minprice\/(\d+)");
+				String MaxPrice = MatchRegexAndReturnValue(path, @"maxprice\/(\d+)");
+				String aboutUs = MatchRegexAndReturnValue(path, @"aboutus");
```

Comment:
The coding standard for this project (which I believe aligns with the .NET framework standard) is Pascal for everything except parameters and local variables.  Interestingly, I went to look this up and it appears that MS forgot to include local variables in their style guidelines.  :D  https://msdn.microsoft.com/en-us/library/ms229043(v=vs.110).aspx  That being said, I don't follow all of their guidelines so don't take that as a source of truth.  :)  As a point of reference, here is .NET Core Framework source code, which uses camelCase local variables: https://github.com/dotnet/corefx/blob/master/src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Binder.cs#L35